routers/init.go

```diff
@@ -44,21 +44,25 @@ func NewServices() {
 }
 
 // In case of problems connecting to DB, retry connection. Eg, PGSQL in Docker Container on Synology
-func initDBEngine() (err error) {
+
+func initDBEngine() (err error, terminal bool) {
 	log.Info("Beginning ORM engine initialization.")
+	var im_err error
 	for i := 0; i < setting.DBConnectRetries; i++ {
 		log.Info("ORM engine initialization attempt #%d/%d...", i+1, setting.DBConnectRetries)
-		if err := models.NewEngine(migrations.Migrate); err == nil {
+		if err = models.NewEngine(migrations.Migrate); err == nil {
```

Comment:
This is the important line

Reply:
Correct.  I left the additional change to give you a chance to take or leave.  It just adds the showing - at the default info log level - the last intermediate failures such that someone does not need to set log level to debug to see the intermediate failures.  I will rewind these for you.

Reply:
It's too complex I think... Just feels messy.